### DISCUSSION TITLE
Glass effect with blur

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#17261] [Q&A]
Has anyone ever tried to create an effect like this? I mean the background effect of the rounded border.
![grafik](https://github.com/user-attachments/assets/ac154cb9-fe19-4331-8bc7-858a3d549c1f)
In Figma this is called glass effect with blur.
![grafik-1](https://github.com/user-attachments/assets/310ee8f0-b682-45f0-824c-b3211abbfd32)

Seems like that may be doable with custom Skia drawing. I wonder whether anyone has a sample for something like this.

**Comment by jmacato** [comment]
Hi @llfab 

There's a control made by @kekekeks and updated by rockdanister here that does that:

https://github.com/rocksdanister/weather/blob/200d11ac9599ae10887d95175d5cd37a4046ea11/src/Drizzle.UI.Avalonia/UserControls/BackdropBlurControl.cs
